style.css

```diff
@@ -0,0 +1,624 @@
+@import url('https://fonts.googleapis.com/css2?family=Merriweather:ital,wght@0,300;0,400;0,700;0,900;1,300;1,400;1,700;1,900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
+
+/*HEX-codes:*/
+/*- Paars `#5722CD`;*/
+/*- Blauw `#4845E4`;*/
+/*- Roze `#ED4385`;*/
+/*- Zwart `#0C0C0C` (voor standaard tekst);*/
+/*- Lichtpaars `#F2F0FF` (voor de grote zwevende woorden, zoals "hello" en "services")*/
+/**Uitzonderingen:**/
+/*- Lichtgrijs `#F3F6F9` (voor de achtergrond van de projecten)*/
+/*- Grijs `#7A7A7A` (voor de service-tegel tekst);*/
+
+
+/*////////////////////////
+    Table of Contents
+    1. Global Styles
+    2. Typography
+    3. Layout
+        3.1 Reusables
+        3.2 Other
+    4. Areas
+        4.1 Header
+        4.2 Introduction
+        etc.
+    5. Media Queries
+////////////////////////*/
+
+/* ------------------------------- 1. Global Styles -------------------------------- */
+body {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-size: 20px;
+}
+
+/* ------------------------------- 2. Typography ----------------------------------- */
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+    font-family: Merriweather, Serif;
+}
+
+p {
+    font-family: Roboto, Serif;
+}
+
+li,
+a {
+    font-family: Roboto, Serif;
+    font-weight: normal;
+    cursor: pointer;
+}
+
+
+/* ------------------------------- 3. Layout --------------------------------------- */
+
+/* --------------- 3.1 Reusables ----------------------- */
+
+.outer-content-container {
+    display: flex;
+    width: 100%;
+    justify-content: center;
+}
+
+.inner-content-container {
+    display: flex;
+    flex-direction: column;
+    flex-basis: 1400px;
+    flex-grow: 0;
+    flex-shrink: 1;
+}
+
+main>.inner-content-container {
+    height: auto;
+}
+
+/* --------------- 3.2 Other --------------------------- */
+
+/* ------------------------------- 4. Areas ---------------------------------------- */
+
+/* --------------- 4.1 Header ------------------------- */
+#header {
+    background: hsla(241, 78%, 60%, 1);
+    background: linear-gradient(315deg, hsla(241, 78%, 60%, 1) 0%, hsla(259, 72%, 47%, 1) 100%);
+    background: -moz-linear-gradient(315deg, hsla(241, 78%, 60%, 1) 0%, hsla(259, 72%, 47%, 1) 100%);
+    background: -webkit-linear-gradient(315deg, hsla(241, 78%, 60%, 1) 0%, hsla(259, 72%, 47%, 1) 100%);
+    filter: progid: DXImageTransform.Microsoft.gradient(startColorstr="#4b49e8", endColorstr="#5722cd", GradientType=1);
```

Comment:
Dit is wel hele oude technologie (ik moest dit googlen). Wat mij betreft hoef je dit niet te doen, omdat alle moderne browsers gebruikmaken van de linear-gradient zoals hierboven

Reply:
Het is aangepast naar background:linear-gradient(45deg, #4845E4, #5722CD);